package.json

```diff
@@ -132,44 +130,9 @@
     "url": "https://github.com/pmndrs/zustand/issues"
   },
   "homepage": "https://github.com/pmndrs/zustand",
-  "jest": {
-    "rootDir": ".",
-    "testEnvironment": "jsdom",
-    "preset": "ts-jest/presets/default-esm",
-    "transform": {
-      "^.+\\.(t|j)sx?$": [
-        "ts-jest",
-        {
-          "useESM": true
-        }
-      ]
-    },
-    "extensionsToTreatAsEsm": [
-      ".ts",
-      ".tsx"
-    ],
-    "moduleNameMapper": {
-      "^zustand$": "<rootDir>/src/index.ts",
-      "^zustand/(.*)$": "<rootDir>/src/$1.ts"
-    },
-    "modulePathIgnorePatterns": [
-      "dist"
-    ],
-    "testRegex": "test.(js|ts|tsx)$",
-    "coverageDirectory": "./coverage/",
-    "collectCoverage": true,
-    "coverageReporters": [
-      "json",
-      "html",
-      "text",
-      "text-summary"
-    ],
-    "collectCoverageFrom": [
-      "src/**/*.{js,ts,tsx}",
-      "tests/**/*.{js,ts,tsx}"
-    ]
-  },
   "dependencies": {
+    "@vitest/ui": "^0.30.1",
+    "eslint-plugin-vitest": "^0.1.4",
```

Comment:
I didn't notice when reviewing, but these shouldn't be in dependencies.

Reply:
https://github.com/pmndrs/zustand/pull/1789/commits/a0090b79008d044df78e07b35bfd5574d1382543